pkg/user/tx_client.go

```diff
@@ -190,6 +198,11 @@ func NewTxClient(
 		txTracker:           make(map[string]txInfo),
 		cdc:                 cdc,
 		gasEstimationClient: gasestimation.NewGasEstimatorClient(conn),
+		// By default, the client operates in trusted mode,
+		// which trusts the consensus node to provide accurate account information
+		// without additional verification.
+		// It is advised to use the consensus endpoint that you trust.
```

Comment:
[nit] this comment won't be read by end-users and won't surface in API docs. The target audience for this code comment should be maintainers. 

IMO this information belongs in the CLI or wherever a user is providing the consensus node endpoint.

```suggestion
```